modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-google-cloud-file-create-sc_{context}"]
+= Creating a storage class for Google Filestore Storage
+
+After installing the operator, you should create a storage class for dynamic provisioning of Google Filestore volumes. 
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+. Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+kind: StorageClass
```

Comment:
@bertinatto can you please help @lpettyjo clean up the storage class? thx!

Reply:
This is better:
```
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: filestore-csi
provisioner: filestore.csi.storage.gke.io
parameters:
  network: network-name <1>
allowVolumeExpansion: true
volumeBindingMode: WaitForFirstConsumer
```
I believe `filestore-csi` is a better name.

Reply:
Done. Thanks!